pkgs/by-name/mv/mvoice/package.nix

```
@@ -0,0 +1,77 @@
+{
+  lib,
+  stdenv,
+  pkgs,
+  gettext,
+  fltk-minimal,
+  alsa-lib,
+  opendht,
+  msgpack-cxx,
+  fmt,
+  gnutls,
+  nlohmann_json,
+  curl,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+stdenv.mkDerivation {
+  pname = "mvoice";
+  version = "0-unstable-2025-09-19";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "n7tae";
+    repo = "mvoice";
+    rev = "aef3662ee6ef86ecb10a71dd686ca41a067162c0";
+    hash = "sha256-FosfD1949shlPXq594gdqodlXM3cGKpFi68tc+/YQ4g=";
+  };
+
+  buildInputs = [
+    gettext
+    fltk-minimal
+    alsa-lib
+    opendht
+    msgpack-cxx
+    fmt
+    gnutls
+    nlohmann_json
+    curl
+    copyDesktopItems
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "mvoice";
+      desktopName = "MVoice";
+      comment = "MVoice by N7TAE - M17 figital voice gateway";
+      exec = "mvoice";
+      categories = [
+        "Network"
+        "AudioVideo"
+        "X-Ham Radio"
+      ];
+    })
+  ];
+
+  preInstall = ''
+    # fix hardcoded paths in Makefile
+    substituteInPlace Makefile --replace "/bin/cp" "cp"
+    substituteInPlace Makefile --replace "/bin/rm" "rm"
```

```suggestion
    substituteInPlace Makefile \
      --replace "/bin/cp" "cp" \
       --replace "/bin/rm" "rm"
```

But also, is the `rm` fix even needed? To me it looks like this is only for *uninstall*, which is not invoked.